geometries/GeoDriver.py

```diff
@@ -15,7 +14,7 @@
 mkfile = './geometries/kernels/mk/asperaMetaKernelM82.tm'
 sp.furnsh(mkfile)
 
-Target = input("Enter target (ASPERA / HST): ")
+Target = 'ASPERA'
```

Comment:
Dropped HST test cases, as HST SPICE kernels have been removed from the kernel set.